lib/Serialization/DeserializeSIL.cpp

```diff
@@ -192,6 +193,32 @@ SILDeserializer::readFuncTable(ArrayRef<uint64_t> fields, StringRef blobData) {
                                                 base + sizeof(uint32_t), base));
 }
 
+Type SILDeserializer::getType(SILBuilder &Builder, ModuleFile *MF,
+                              serialization::TypeID TID) {
+  auto Ty = MF->getType(TID);
+  if (Ty && Ty->hasOpenedExistential()) {
```

Comment:
Why not invert this condition and early exit. It eliminates a level of indentation.